docker/checkout.bash

```diff
@@ -33,16 +34,9 @@ while [[ $# -gt 0 ]]; do
 done
 
 cd ${dir}/autoware.ai
-
-if [[ "$BRANCH" = "develop" ]]; then
-      git clone --depth=1 https://github.com/usdot-fhwa-stol/carma-msgs.git --branch $BRANCH
-      git clone --depth=1 https://github.com/usdot-fhwa-stol/carma-utils.git --branch $BRANCH
-      git clone --depth=1 https://github.com/usdot-fhwa-stol/autoware.auto.git --branch $BRANCH
-else
-      git clone --depth=1 https://github.com/usdot-fhwa-stol/carma-msgs.git --branch develop
-      git clone --depth=1 https://github.com/usdot-fhwa-stol/carma-utils.git --branch develop
-      git clone --depth=1 https://github.com/usdot-fhwa-stol/autoware.auto.git --branch develop
-fi
-
+git clone --depth=1 https://github.com/usdot-fhwa-stol/carma-msgs.git --branch $BRANCH
+git clone --depth=1 https://github.com/usdot-fhwa-stol/carma-utils.git --branch $BRANCH
+git clone --depth=1 https://github.com/usdot-fhwa-stol/autoware.auto.git --branch $BRANCH
+# TODO: Should this external dependencies be moved into install.sh
```

Comment:
Similar to previous comment, let's add a JIRA story number here for tracking (and create one if one hasn't been created yet).

Reply:
https://usdot-carma.atlassian.net/browse/CAR-6023